Client/Frontend/Strings.swift

```diff
@@ -1274,35 +1274,17 @@ extension String {
                 value: "Notifications",
                 comment: "In the settings menu, in the Privacy section, this is the title for Notifications customization section."
             )
-            public static let AllowAllNotificationsTitle = MZLocalizedString(
-                "Settings.Notifications.AllowAllNotificationsTitle",
+            public static let SyncNotificationsTitle = MZLocalizedString(
+                "Settings.Notifications.SyncNotificationsTitle",
                 tableName: "Settings",
-                value: "Allow All Notifications",
-                comment: "This is the title for the setting that toggles All Notifications feature in the settings menu under the Notifications section."
+                value: "Sync",
+                comment: "This is the title for the setting that toggles Sync related notifications in the settings menu under the Notifications section."
             )
-            public static let TabsNotificationsTitle = MZLocalizedString(
-                "Settings.Notifications.TabsNotificationsTitle",
+            public static let SyncNotificationsStatus = MZLocalizedString(
+                "Settings.Notifications.SyncNotificationsStatus",
```

Comment:
I think we can add a `.v112` to the string keys as well.

Reply:
Oh yes, absolutely. Totally missed that. Thanks!